test/shapirowilk.jl

```diff
@@ -0,0 +1,70 @@
+@test SWCoeffs(3).A == [sqrt(2.0)/2.0]
+@test SWCoeffs(3).N == 3
+
+S = SWCoeffs(10)
+@test S.N == 10
+@test length(S) == 10
+@test lastindex(S) == 10
+@test length(S.A) == 5
+
+@test S.A[4] == S[4] == -S[7]
+@test S.A[2] == S[2] == -S[9]
+
+S = SWCoeffs(11)
+@test S.N == 11
+@test length(S) == 11
+@test lastindex(S) == 11
+@test length(S.A) == 5
+
+@test S[6] == 0.0
+@test S.A[5] == S[5] == -S[7]
+@test S.A[3] == S[3] == -S[9]
+@test S.A[1] == S[1] == -S[11]
+
+#anti-symmetry
+S = SWCoeffs(20)
+@test all([S[i] == -S[end-i+1] for i in 1:length(S)])
+
+
+# Values obtained by calling `_swilkfort` fortran subroutine directly.
+
+SWc10 = SWCoeffs(10)
+a = SWc10.A .- [0.573715, 0.32897, 0.214349, 0.122791, 0.0400887]
+@test norm(a,1) ≈ 0.0 atol=S.N*eps(Float32)
+
+SWc20 = SWCoeffs(20)
+b = SWc20.A .- [0.473371, 0.32174, 0.255663, 0.208297, 0.16864, 0.133584, 0.101474, 0.0712893, 0.0423232, 0.0140351]
+@test norm(b,1) ≈ 0.0 atol=S.N*eps(Float32)
+
+if VERSION >= v"1.8"
```

Comment:
Why does this require Julia 1.8? AFAICT it's not relying on any version-specific features. If it's for random number stability, it'd be better to use StableRNGs.jl.

Reply:
Yes, if I recall correctly this was about rng. that should be done by now :)